.tekton/linter.yaml

```diff
@@ -125,6 +125,7 @@ spec:
               image: quay.io/eclipse/che-docs
               workingDir: $(workspaces.source.path)
               script: |
+                export NO_COLOR=1
```

Comment:
I see in the description that we did not get coloring for Openshift console the why we need to disable it :thinking:

Reply:
it doesn't parse terminal colours characters to be translated as html colours, i logged an issue for console a few years back about it but haven't heard anything from them fixing it.

Reply:
ah i see got it